src/libcore/tests/hash/sip.rs

```diff
@@ -249,12 +249,21 @@ fn test_hash_usize() {
     assert!(hash(&(val as u64)) != hash(&(val as usize)));
     assert_eq!(hash(&(val as u32)), hash(&(val as usize)));
 }
-#[test] #[cfg(target_arch = "x86_64")]
+
+#[test] #[cfg(all(target_arch = "x86_64", target_pointer_width = "32"))]
```

Comment:
We've got existing tests for x86/x86_64, perhaps those could just switch to being two tests based on target_pointer_width?

Reply:
Done.